modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/near/GridNearUnlockRequest.java

```diff
@@ -85,11 +139,25 @@ public GridNearUnlockRequest(int cacheId, int keyCnt, boolean addDepInfo) {
 
     /** {@inheritDoc} */
     @Override public byte fieldsCount() {
-        return 8;
+        return 11;
```

Comment:
Why 11 here? I see only 2 fileds added, 8+2 == 10 :)

Reply:
+1 in `GridDistributedUnlockRequest`.